app/shared/user-agent-http-client.js

```diff
@@ -73,6 +74,9 @@ export class UserAgentHttpClient {
       // does not have its own sessionId, we probably hit a race condition,
       // so wait until the page has its sessionId by listening for this promise.
       return this.PAGE_TO_SESSION.get(page.id);
+    } else if (UUIDUtil.isUUID(page) && this.PAGE_TO_SESSION.has(page)) {
+      // Also allow passing in the pageId itself
+      return this.PAGE_TO_SESSION.get(page);
```

Comment:
I think most of this logic is from the original implementation months ago, unsure if all these forks are still necessary/used